.changeset/odd-icons-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+'style-dictionary': patch
```

Comment:
Docs changes don't need a changeset, right @jorenbroekema ? We don't actually need to bump the npm package to change the docs

Reply:
True but a small type adjustment was done inside the library code itself which does require a patch bump so users get the correct autocomplete for when they use those formats functions e.g. `StyleDictionary.hooks.formats['ios-swift/class.enum'].format({})` -> needs to autocomplete the options.className

Reply:
Ah you are right, we should change the commit message on merge then to reflect that its not just a docs change